pkgs/top-level/all-packages.nix

```diff
@@ -3804,6 +3804,160 @@ let
+
+    ocaml_xcp-rrd = callPackage ../development/ocaml-modules/ocaml-xcp-rrd { };
+
+    ocaml_xenstore = callPackage ../development/ocaml-modules/ocaml-xenstore { };
+
+    ocaml_core = callPackage ../development/ocaml-modules/ocaml-core { };
+
+    ocaml_backtrace = callPackage ../development/ocaml-modules/ocaml-backtrace { };
+
+    ocaml_evtchn = callPackage ../development/ocaml-modules/ocaml-evtchn { xen = xen_xenServer; };
+
+    ocaml_gnt = callPackage ../development/ocaml-modules/ocaml-gnt { xen = xen_xenServer; };
+
+    ocaml_mirage-types = callPackage ../development/ocaml-modules/ocaml-mirage-types { };
+
+    ocaml_xenstore-clients = callPackage ../development/ocaml-modules/ocaml-xenstore-clients { };
+
+    ocaml_async-kernel = callPackage ../development/ocaml-modules/ocaml-async-kernel { };
+
+    ocaml_mirage = callPackage ../development/ocaml-modules/ocaml-mirage { };
+
+    ocaml_mirage-clock-unix = callPackage ../development/ocaml-modules/ocaml-mirage-clock-unix { };
+
+    ocaml_mirage-clock-xen = callPackage ../development/ocaml-modules/ocaml-mirage-clock-xen { };
+
+    ocaml_vchan = callPackage ../development/ocaml-modules/ocaml-vchan { xen = xen_xenServer; };
+
+    ocaml_vhd = callPackage ../development/ocaml-modules/ocaml-vhd { };
+
+    ocaml_async-unix = callPackage ../development/ocaml-modules/ocaml-async-unix { };
+
+    ocaml_mirage-xen = callPackage ../development/ocaml-modules/ocaml-mirage-xen { };
+
+    ocaml_async-extra = callPackage ../development/ocaml-modules/ocaml-async-extra { };
+
+    ocaml_mirage-block-xen = callPackage ../development/ocaml-modules/ocaml-mirage-block-xen { };
+
+    ocaml_mirage-console-xen = callPackage ../development/ocaml-modules/ocaml-mirage-console-xen { };
+
+    ocaml_async = callPackage ../development/ocaml-modules/ocaml-async { };
+
+    ocaml_async-find = callPackage ../development/ocaml-modules/ocaml-async-find { };
+
+    ocaml_conduit = callPackage ../development/ocaml-modules/ocaml-conduit { };
+
+    ocaml_textutils = callPackage ../development/ocaml-modules/ocaml-textutils { };
+
+    ocaml_async-inotify = callPackage ../development/ocaml-modules/ocaml-async-inotify { };
+
+    ocaml_cohttp = callPackage ../development/ocaml-modules/ocaml-cohttp { };
+
+    ocaml_testvmlib = callPackage ../development/ocaml-modules/ocaml-testvmlib { xen = xen_xenServer; };
+
+    ocaml_xen-api-client = callPackage ../development/ocaml-modules/ocaml-xen-api-client { };
+
+    ocaml_rrd-transport = callPackage ../development/ocaml-modules/ocaml-rrd-transport { xen = xen_xenServer; };
+
+    ocaml_xen-api-libs-transitional = callPackage ../development/ocaml-modules/ocaml-xen-api-libs-transitional { xen = xen_xenServer; };
+
+    ocaml_rrdd-plugin = callPackage ../development/ocaml-modules/ocaml-rrdd-plugin { xen = xen_xenServer; };
+
+    ocaml_xenops = callPackage ../development/ocaml-modules/ocaml-xenops { xen = xen_xenServer; };
+
```

Comment:
Minor nit: prefixing all of these names with ```ocaml_``` seems redundant, since they're already inside ```mkOcamlPackages```. Also it is inconsistent with the surrounding names.